src/tests/tracing/eventpipe/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Except for a targeted testing, the full Tracing test tree is not yet ready for NativeAOT-->
+    <NativeAotIncompatible>true</NativeAotIncompatible>
```

Comment:
Random disablement/enablement with `NativeAotIncompatible` in project files/props files runs the risk that we'll never enable something because it will be forgotten. Issues.targets is the mechanism to temporarily disable tests that keeps everything in a single spot. We should have one group for EventPipe in there.